feature-configs/5g-ref-nw/profile-base/cu-up/sriov/sriov-nnp-cu-up-numa0-bh-dpdk.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-nnp-cu-up-numa0-bh-u
+  namespace: openshift-sriov-network-operator
+spec:
+# This works for Intel based NICs.
+# For Mellanox please change to:
+#     deviceType: netdevice
+#     isRdma: true
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+# The exact physical function name must match the hardware used    
+    pfNames: ["ens1f0"]
```

Comment:
Since this is a template that *must* be specified with the right physical device name, perhaps the profile-base should have this commented out?

Reply:
Good point, thanks

Reply:
@lack 
Actually after rethinking this, since we are going to certify hardware platforms, NIC device names will be the same in all the servers of the same platform. The only thing we are supposed to customize is VLANs. That's why I initially fixed the device names in the profile, and VLANs in the customizations.
Please tell me whether it makes more sense now

Reply:
@vitus133 see
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/consistent-network-interface-device-naming_configuring-and-managing-networking
that means that certification will need to dictate what PCIe slots to use (otherwise names will be different)
so are we really dictating slot 1?

also, I guess this call for documentation (in README.md?) that this is a single port (no bonding) configuration, always using port#0 of the NIC. you are also dictating that